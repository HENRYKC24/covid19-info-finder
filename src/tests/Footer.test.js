import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('Navbar renders the UI as expected', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
