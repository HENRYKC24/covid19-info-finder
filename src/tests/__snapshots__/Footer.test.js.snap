// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders the UI as expected 1`] = `
<footer
  className="footer"
>
  <p
    className="footer-text"
  >
    Created with ðŸ’– by Henry Kc at Microverse Inc., US.
  </p>
</footer>
`;
